book/opmanual_autolab/50_autolab_operation_manual/51_autolab_HW_checks_script.md

```diff
@@ -0,0 +1,117 @@
+# Autobot hardware compliance checks {#autolab-autobot-hw-checks status=ready}
+
+Excerpt: run a compliance test and store the information at the right place
+
+We have put together a step-by-step instruction to ensure that every Autobot that enters the Autolab is ready to go. We keep track of them in a GitHub Repository which will be used to keep track of the change history of every Autobot as well as to provide additional information to the user.
+
+<div class='requirements' markdown="1">
+
+Requires: a assembled [Autobot](#autolab-autobot-specs)
+
+Results: a Autobot which is ready to be used in the Autolab.
+
+</div>
+
+## Checklist
+
+### Step 0
+
+Power on the Duckiebot. Then run the following command on your Computer. It will launch a Docker container on your Duckiebot which creates the suitability test report:
+
+    laptop $ docker -H HOSTNAME.local run -it --network host -v /data:/data -v /sys/firmware/devicetree:/devfs:ro duckietown/duckiebot_hw_checks:daffy
+
+### Step 1
+
+If you mounted the Raspberry Pi on top, check that a pin protector is mounted on the pins of the hut to prevent pins from shorting out.
+If no protector is installed yet, then first check that no pins are bent or are making contact and then add the protector. It should slide into place without applying any force. The stl file to 3D print the pin protector can be found in [](#adding-hw-to-bot).
+
+
+### Step 2
+
+Make sure there is a SD card plugged into the Duckiebot for logging using a USB adapter. As the adapter has a LED that might disturb the performance of the other Duckiebots, you need to put a piece of tape on it. Make sure that you cannot see any more light coming from the LED before proceeding. A USB memory stick is fine as well for storage.
```

Comment:
please remove or make this part optionnal, as we dont use that anymore.

Reply:
removed